ui/components/Dream/Sidebar.js

```diff
@@ -105,10 +123,10 @@ const DreamSidebar = ({ dream, event, currentOrgMember, canEdit }) => {
       },
     ],
   });
-
-  const [contributeModalOpen, setContributeModalOpen] = useState(false);
-  const [cocreatorModalOpen, setCocreatorModalOpen] = useState(false);
-  const [actionsDropdownOpen, setActionsDropdownOpen] = useState(false);
+  const [editTags] = useMutation(EDIT_TAGS_MUTATION, {
```

Comment:
Right now if i add/remove a tag here and go back to the collection page and search, the results will be old. I think the fix would be to add an `update` to the cache like here https://github.com/Edgeryders-Participio/multi-dreams/blob/cf711d187b9da0d9d9effe6e2efc7de8d74a6cc1/ui/components/Realities/index.js#L52

Reply:
This will only occur if you go back to an old query for a tag on the collection page without refresh, if you go back and search again it seems to work on my end?

It'd be a pretty tricky to implement an update cache function for this since you would try recreate the results mongodb gives back when querying for specific tags. Removing an item when there are certain tags and adding an item to the cache when there are certain tags, better to let the API handle that.

Reply:
Hmm yeah maybe a better solution is to make the search query trust the cache less? Maybe the cache-and-network policy? https://www.apollographql.com/docs/react/data/queries/#cache-and-network

Reply:
Pushed a fix that sets 'cache-and-network' policy on the dreams query